paris-processor/src/main/java/com/airbnb/paris/processor/models/StyleableInfo.kt

```diff
@@ -101,6 +101,13 @@ internal class StyleableInfo(
     baseStyleableInfo: BaseStyleableInfo
 ) : BaseStyleableInfo(baseStyleableInfo), WithSkyProcessor {
 
+    /**
+     * A styleable declaration is guaranteed to be in the same R file as any attribute or styleable child.
+     * `min` is used to ensure in the case there are multiple R files, a consistent one is chosen.
```

Comment:
If we expect a unique `R` class maybe we should throw if that's not the case? I can't think of a valid use case with different `R` classes, but I can think of some invalid ones that would be good to highlight.

Reply:
We don't necessarily expect a unqiue R value. 

A styleable definition could technically be split across different modules and android will merge the attributes from the two into the final styleable. So a component could validly point to multiple r values from attributes split across modules, but it's fine to use the stylable from either one.

Reply:
I've never encountered a split styleable definition. Are we both talking about `<declare-styleable …>`?

Reply:
Yes. I'm not saying that it is a good or common practice, but it is allowed.

Reply:
Wow. Good to know.

Reply:
@BenSchwab isn't this a problem when namespaced resources are turned on and we're referring to a specific styleable R class here?